crates/biome_css_analyze/tests/specs/nursery/noDuplicateSelectorsKeyframeBlock/invalid.css

```diff
@@ -0,0 +1,5 @@
+@keyframes foo { from {} from {}}
+
+@keyframes foo { from {} FROM {}}
+
+@keyframes foo { 0% {} 0% {}}
```

Comment:
We can add more test cases
https://github.com/stylelint/stylelint/blob/main/lib/rules/keyframe-block-no-duplicate-selectors/__tests__/index.mjs